src/collectd.conf.pod

```diff
@@ -5438,6 +5438,17 @@ behavior is to let the kernel choose the appropriate interface. Be warned
 that the manual selection of an interface for unicast traffic is only
 necessary in rare cases.
 
+=item B<BindAddress> I<IP Address>
+
+Set the outgoing IP address for IP packets. This option can be used instead of
+the I<Interface> option to explicitly define the IP address which will be used
+to send Packets to the remote server. This applies at least
+to IPv6 packets and if possible to IPv4. If this option is not applicable,
```

Comment:
> This applies at least to IPv6 packets and if possible to IPv4

I don't understand this sentence. Why would this not apply to IPv4?

Reply:
Since the BindAddress is the same as the Interface option, I did a copy/paste from the Interface option and modify it according to the usage (IP address instead of Interface). The explanation is the same so I left it as the Interface description. I don't know why the Interface option would not apply to IPv4 as it surely does but since they both do the same (except the type of bind) and someone wrote it for a reason, I didn't want to change it,